diff --git a/extensions/browser/extension_event_histogram_value.h b/extensions/browser/extension_event_histogram_value.h
index 695b0464b4f840a241b04205ecc9ce4059f11f7a..713c873d1dbb42fe4cf46bd7410ac2759cf341a6 100644
--- a/extensions/browser/extension_event_histogram_value.h
+++ b/extensions/browser/extension_event_histogram_value.h
@@ -486,6 +486,7 @@ enum HistogramValue {
   ACCESSIBILITY_PRIVATE_ON_MAGNIFIER_BOUNDS_CHANGED = 464,
   FILE_MANAGER_PRIVATE_ON_PIN_TRANSFERS_UPDATED = 465,
   ACCESSIBILITY_PRIVATE_ON_POINT_SCAN_SET = 466,
+  BRAVE_HISTOGRAM_VALUES
   // Last entry: Add new entries above, then run:
   // python tools/metrics/histograms/update_extension_histograms.py
   ENUM_BOUNDARY
